src/proposals/20231115/SparkEthereum_20231115.t.sol

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../../SparkTestBase.sol';
+
+import { Domain, GnosisDomain } from 'xchain-helpers/testing/GnosisDomain.sol';
+
+import { SparkEthereum_20231115 } from './SparkEthereum_20231115.sol';
+import { SparkGnosis_20231115 } from   './SparkGnosis_20231115.sol';
+
+interface IL2BridgeExecutor {
+    function execute(uint256 index) external;
+}
+
+interface D3MHubLike {
+    function exec(bytes32) external;
+}
+
+contract SparkEthereum_20231115Test is SparkEthereumTestBase {
+
+    address public constant GNOSIS_BRIDGE_EXECUTOR = 0xc4218C1127cB24a0D6c1e7D25dc34e10f2625f5A;
+    address public constant D3M_JOB                = 0x1Bb799509b0B039345f910dfFb71eEfAc7022323;
+    address public constant D3M_HUB                = 0x12F36cdEA3A28C35aC8C6Cc71D9265c17C74A27F;
+
+    bytes32 public constant DAI_ILK = 0x4449524543542d535041524b2d44414900000000000000000000000000000000;
+
+    Domain       mainnet;
+    GnosisDomain gnosis;
+
+    constructor() {
+        id = '20231115';
+    }
+
+    function setUp() public {
+        mainnet = new Domain(getChain('mainnet'));
+        gnosis = new GnosisDomain(getChain('gnosis_chain'), mainnet);
+
+        gnosis.selectFork();
+        // This will be removed after spell deployment
+        new SparkGnosis_20231115();
+
+        mainnet.selectFork();
+
+        payload = deployPayload();
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+    function testSpellSpecifics() public {
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+
+        /*****************************************/
+        /*** rETH Supply Cap Before Assertions ***/
+        /*******************************************/
+
+        ReserveConfig memory rETHConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'rETH');
+        assertEq(rETHConfigBefore.supplyCap, 60_000);
+
+        /*******************************************/
+        /*** wstETH Supply Cap Before Assertions ***/
+        /*******************************************/
+
+        ReserveConfig memory wstETHConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'wstETH');
+        assertEq(wstETHConfigBefore.supplyCap, 400_000);
+
+        /*********************************/
+        /*** DAI LTV Before Assertions ***/
+        /*********************************/
+
+        ReserveConfig memory DAIConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'DAI');
+        assertEq(DAIConfigBefore.ltv, 1);
+
+        /*****************************************************/
+        /*** WETH Interest Rate Strategy Before Assertions ***/
+        /*****************************************************/
+
+        ReserveConfig memory wethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'WETH');
+        IDefaultInterestRateStrategy interestRateStrategy = IDefaultInterestRateStrategy(
```

Comment:
Lets update this to `oldInterestStrategy` just to make it super clear that its the before state being queried in the assertions below